tests/integration/go_ethereum/test_goethereum_http.py

```diff
@@ -57,7 +57,7 @@ def _geth_command_arguments(rpc_port, base_geth_command_arguments, geth_version)
             "--http.port",
             rpc_port,
             "--http.api",
-            "admin,eth,net,web3,personal,miner,txpool",
+            "admin,eth,net,web3,personal,txpool",
```

Comment:
pretty sure we can remove `personal` from here as well

Reply:
We still utilize deprecated personal stuff for now so tests require it.

Reply:
Which tests though because we don't need it. `geth --dev` unlocks the coinbase account so we can possibly use that for funding, etc.